go/test/endtoend/vtgate/foreignkey/fk_fuzz_test.go

```diff
@@ -604,61 +608,65 @@ func TestFkFuzzTest(t *testing.T) {
-		maxValForCol:   5,
-		maxValForId:    10,
-		insertShare:    100,
-		deleteShare:    0,
-		updateShare:    0,
-	}, {
-		name:           "Single Thread - Balanced Inserts and Deletes",
-		concurrency:    1,
-		timeForTesting: 5 * time.Second,
-		maxValForCol:   5,
-		maxValForId:    10,
-		insertShare:    50,
-		deleteShare:    50,
-		updateShare:    0,
-	}, {
-		name:           "Single Thread - Balanced Inserts and Updates",
-		concurrency:    1,
-		timeForTesting: 5 * time.Second,
-		maxValForCol:   5,
-		maxValForId:    10,
-		insertShare:    50,
-		deleteShare:    0,
-		updateShare:    50,
-	}, {
-		name:           "Single Thread - Balanced Inserts, Updates and Deletes",
-		concurrency:    1,
-		timeForTesting: 5 * time.Second,
-		maxValForCol:   5,
-		maxValForId:    10,
-		insertShare:    50,
-		deleteShare:    50,
-		updateShare:    50,
-	}, {
-		name:           "Multi Thread - Only Inserts",
-		concurrency:    30,
-		timeForTesting: 5 * time.Second,
-		maxValForCol:   5,
-		maxValForId:    30,
-		insertShare:    100,
-		deleteShare:    0,
-		updateShare:    0,
-	}, {
-		name:           "Multi Thread - Balanced Inserts, Updates and Deletes",
-		concurrency:    30,
-		timeForTesting: 5 * time.Second,
-		maxValForCol:   5,
-		maxValForId:    30,
-		insertShare:    50,
-		deleteShare:    50,
-		updateShare:    50,
-	}}
+	}{
+		//{
+		//	name:           "Single Thread - Only Inserts",
+		//	concurrency:    1,
+		//	timeForTesting: 5 * time.Second,
+		//	maxValForCol:   5,
+		//	maxValForId:    10,
+		//	insertShare:    100,
+		//	deleteShare:    0,
+		//	updateShare:    0,
+		//}, {
+		//	name:           "Single Thread - Balanced Inserts and Deletes",
+		//	concurrency:    1,
+		//	timeForTesting: 5 * time.Second,
+		//	maxValForCol:   5,
+		//	maxValForId:    10,
+		//	insertShare:    50,
+		//	deleteShare:    50,
+		//	updateShare:    0,
+		//}, {
+		//	name:           "Single Thread - Balanced Inserts and Updates",
+		//	concurrency:    1,
+		//	timeForTesting: 5 * time.Second,
+		//	maxValForCol:   5,
+		//	maxValForId:    10,
+		//	insertShare:    50,
+		//	deleteShare:    0,
+		//	updateShare:    50,
+		//},
+		{
+			name:           "Single Thread - Balanced Inserts, Updates and Deletes",
+			concurrency:    1,
+			timeForTesting: 5 * time.Second,
+			maxValForCol:   5,
+			maxValForId:    10,
+			insertShare:    50,
+			deleteShare:    50,
+			updateShare:    50,
+		},
+		//{
+		//	name:           "Multi Thread - Only Inserts",
+		//	concurrency:    30,
+		//	timeForTesting: 5 * time.Second,
+		//	maxValForCol:   5,
+		//	maxValForId:    30,
+		//	insertShare:    100,
+		//	deleteShare:    0,
+		//	updateShare:    0,
+		//}, {
+		//	name:           "Multi Thread - Balanced Inserts, Updates and Deletes",
+		//	concurrency:    30,
+		//	timeForTesting: 5 * time.Second,
+		//	maxValForCol:   5,
+		//	maxValForId:    30,
+		//	insertShare:    50,
+		//	deleteShare:    50,
+		//	updateShare:    50,
+		//},
```

Comment:
same here
```suggestion
```

Reply:
oversight after testing. will put it back